src/app/core/mpi/components/paciente-cru.component.ts

```diff
@@ -581,8 +583,18 @@ export class PacienteCruComponent implements OnInit {
                 pacienteGuardar.direccion[0].ubicacion.localidad = this.localidadActual;
             }
 
-            this.pacienteService.save(pacienteGuardar, ignoreCheck, false).subscribe(
-                (resultadoSave: any) => {
+            let guardar;
+            if (this.pacienteOld) {
+                const savePacienteOld = this.pacienteService.post(this.pacienteOld, true);
+                const savePacienteNew = this.pacienteService.save(pacienteGuardar, ignoreCheck, false);
+                guardar = concat(savePacienteNew, savePacienteOld);
+            } else {
+                guardar = this.pacienteService.save(pacienteGuardar, ignoreCheck, false);
+            }
+
```

Comment:
Yo lo haría de la siguiente manera:
const savePacienteNew = this.pacienteService.save(pacienteGuardar, ignoreCheck, false); 
// Ya que lo vas a hacer si o si.
Luego pregunto si tengo pacienteOld
 if (this.pacienteOld) {
                const savePacienteOld = this.pacienteService.post(this.pacienteOld, true);
                guardar = concat(savePacienteNew, savePacienteOld);
}
y sigo normal...